pkg/ebpftracer/tracer_test.go

```diff
@@ -162,6 +164,16 @@ func TestTracer(t *testing.T) {
 	}
 }
 
+func prettyPrintProtobuf(msg protoreflect.ProtoMessage) string {
```

Comment:
Isn't this technically `prettySprintProtobuf` since it just returns a formatted string instead of printing it directly?